commands/system.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+var SystemCmd = &cobra.Command{
+	Use:   "system",
+	Short: "System management",
```

Comment:
Can we add a `Long` description here?

Reply:
Fixed via https://github.com/mattermost/mmctl/pull/130/commits/62adb429e8db58d66c95d6dd13a8bd538fe46bd7